css/landing-page.css

```diff
@@ -186,6 +186,22 @@ header.masthead h1 {
     margin-left: 6rem;
   }
 }
+@media (min-width: 0px) and (max-width: 860px){

```

Comment:
I am not sure if we really need the min width here if you set it to 0.

Reply:
Fixed!